.github/workflows/lint.yml

```diff
@@ -34,23 +34,22 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      # NOTE: The latest version of staticcheck specifies "go 1.23" in its go.mod
-      # file, but when checking Go 1.24 code, needs to be built with Go 1.24. Use the
-      # pre-built binaries instead, which are built with Go 1.24.
+      # NOTE: The latest version of staticcheck specifies "go 1.24.1" in its go.mod
```

Comment:
I believe this updated comment is incorrect, as the latest version of staticcheck still [specifies](https://github.com/dominikh/go-tools/blob/2025.1.1/go.mod#L3) "go 1.23" in its go.mod. I don't really understand why we can now use `go install` and we couldn't before?

Either way, maybe it's best just to keep installing staticcheck from binaries, remove the commented-out lines, and replace the big block comment with "Install staticcheck from binary to avoid go.mod issues".

Reply:
Ah, got it, and thanks for the comment fix.